bitbots_move_base/config/dwa_local_planner_config.yaml

```diff
@@ -28,20 +28,20 @@ DWAPlannerROS:
   
   
   # dwa specific
-  sim_time: 2                     # The amount of time to roll trajectories out for in seconds
-  sim_granularity: 0.025            # The granularity with which to check for collisions along each trajectory in meters
+  sim_time: 5                     # The amount of time to roll trajectories out for in seconds
+  sim_granularity: 0.25            # The granularity with which to check for collisions along each trajectory in meters
```

Comment:
should this be larger than the width of a goalpost/player? Seems a bit dangerous to me.

Reply:
good point